src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -40,7 +40,7 @@ public static void SetStatus(this Activity activity, Status status)
         {
             Debug.Assert(activity != null, "Activity should not be null");
 
-            activity.SetTag(SpanAttributeConstants.StatusCodeKey, (int)status.StatusCode);
+            activity.SetTag(SpanAttributeConstants.StatusCodeKey, StatusHelper.GetStringNameForStatusCode(status.StatusCode));
```

Comment:
can you add the type change in changelog.md for API?

Reply:
Updated